cspell.config.yaml

```diff
@@ -29,6 +29,7 @@ ignorePaths:
   - mkdocs.yml
   - fathom.html
   - _vendor
+  - go
```

Comment:
Why is the go sdk exempt from spell checking?

Reply:
Go programs are apparently not written in English 🤷🏼